anotherfile.js

```diff
@@ -0,0 +1,15 @@
+function calculate(operation, a, b) {
+  switch (operation) {
+    case "add":
+      return a + b;
+    case "subtract":
+      return a - b;
+    case "multiply":
+      return a * b;
+    case "divide":
+      if (b === 0) return "Error: Cannot divide by zero";
```

Comment:
The function should consistently return a number or throw an error instead of returning a string for error cases.

```suggestion
if (b === 0) throw new Error("Cannot divide by zero");
```

**Original code:**
```
if (b === 0) return "Error: Cannot divide by zero";
```